docker-compose.yaml

```
@@ -133,7 +140,7 @@ x-medusa-base: &medusa-base
   networks:
     - internal
   environment:
-    <<: *medusa-env
+    <<: [ *medusa-env, *medusa-email-env ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)